### DISCUSSION TITLE
support.github.com is translated to non-English without being able to get back to English

### DISCUSSION THREAD

**Discussion by jord1e** [community/community#77053] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

Hello, I am a foreign language learner and when I open https://support.github.com/ it automatically "translates" to German, because German is a supported support language.

I vastly prefer my support experience to be in English, even though I can read the German support documentation.

Currently, I have to open the support page in a browser with German UA disabled, which is quite a hassle.

**_Please make it possible to switch the language on https://support.github.com/_**

Ps. Translation in the footer are missing
![image](https://github.com/community/community/assets/30464310/e6942c22-d8bb-4874-81d1-a1affa47a237)

**Comment by jrom99** [comment]
Same here, my account is set to English, but my browser to Brazilian Portuguese. This creates a mismatch in the interface. Most domain-specific tech terms make little sense when translated to Portuguese (e.g.: "Detach fork" vs "Separar uma bifurcação"), so I actually cannot read/understand the support documentation. Additionally, in order to create a support ticket, I had to do lots of guessing based on the English parts (ticket subject, URL, etc.).

<img width="808" height="369" alt="image" src="https://github.com/user-attachments/assets/e7d17182-e8fa-4ff7-9e72-d4780495174d" />
